### PR TITLE
Add mridc

### PR DESCRIPTION
**Pull request by wdika**
## Before submitting

- [x] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
- [X] Did you create/update your **configuration file**?
- [X] Did you **add your config to CI** - GitHub action and Azure pipeline?
- [ ] Are all integration **tests passing**?

## What does this PR do? \[optional\]

Fixes # (issue).

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -18,3 +18,4 @@
 /configs/PyTorchLightning/flash*    @ethanwharris
 /configs/PyTorchLightning/lightning_transformers*    @seannaren
 /configs/SeanNaren/*    @seannaren
+/configs/wdika/mridc/*    @wdika
```
Comment (Borda) [inline .github/CODEOWNERS:21]:
```suggestion
/configs/wdika/*    @wdika
```

File: configs/wdika/mridc-release.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+dependencies:
+  - name: pytorch-lightning
+    HTTPS: https://github.com/PyTorchLightning/pytorch-lightning.git
+    checkout: release/1.5.x
+    install_extras: loggers
+
+testing:
+  # additional pytest arguments
+  pytest_args: -m "not pleasefixme and not torch_tts" --cpu --with_downloads --relax_numba_compat
```
Comment (Borda) [inline configs/wdika/mridc-release.yaml:22]: is this really needed?

Comment (wdika) [reply configs/wdika/mridc-release.yaml:22]: Hi @Borda, thanks for the quick review. My backend is based on NVIDIA's NeMO and I do want to test numba compatibility as well. Do you think it's not needed?

File: configs/wdika/mridc-release.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+target_repository:
+  HTTPS: https://github.com/wdika/mridc
+  # checkout a particular branch or a tag
+  checkout: main
+  # define installing package extras
+  install_extras: all
```
Comment (Borda) [inline configs/wdika/mridc-release.yaml:6]:
```suggestion
```
seems your setup does not include extra...

**Review by wdika — COMMENTED** [review]
Updated according to suggestions from code review.

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
Seems that the given test arguments are not valid:
```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --cpu --with_downloads --relax_numba_compat
```

**Comment by wdika** [comment]
@Borda I would prefer this to be closed. It has been open for months and I keep getting nonsense notifications. I have covered my repo with CI tests independently of this. Thanks for the past help.

**Comment by Borda** [comment]
@wdika ok, as you wish, just would like to apology see that it took so long time... :(
 eventually, I will have some time next week... :)

**Pull request closed without merging**